feedback.md

```diff
@@ -0,0 +1,37 @@
+AMBULO
+======
+<br />
+
+| Team Member | Role                 |
+| ------------| -------------------- |
+| Anton Zheng | Dev Lead             |
+| Jenny Yang  | Repo Master          |
+| Karen Lee   | User Experience Lead |
+| Thoa Nguyen | Test Lead/PM         |
+
+<br />
+
+## DESCRIPTION
+A hiking web-based application that uses Trail API for locations and Instagram API for images of trails. 
```

Comment:
Sounds like, currently, the main focus or the most challenging part of building this application is using the response from the Trail API to search the Instagram API. Before going on, you should test out the APIs to make sure that they are compatible. Many times, you will need to do a lot of modification to a response to use it to be compatible with other API's.